Libs/MRML/Core/Testing/vtkMRMLSceneImportIDModelHierarchyConflictTest.cxx

```diff
@@ -166,7 +166,7 @@ int ImportIDModelHierarchyConflictTest()
   //               |-- ref [displayNodeID] to vtkMRMLModelDisplayNode4
   //               |-- ref [associatedNodeRef] to vtkMRMLModelNode2
```

Comment:
Should comment like the one above be updated ?

Reply:
Pushed update that adds comments